### PR TITLE
Update jest to ~21.2.1

### PR DESCRIPTION
**Pull request by nickpape-msft**

### CODE REVIEW

**Comment by iclanton** [comment]
Approved.

File: apps/api-documenter/package.json [hunk 1]
```diff
@@ -29,6 +29,6 @@
     "@types/js-yaml": "3.9.1",
     "@types/node": "6.0.88",
     "gulp": "~3.9.1",
-    "@types/jest": "~20.0.4"
+    "@types/jest": "~21.1.8"
```
Comment (qz2017) [inline apps/api-documenter/package.json]: Did you test the case when one project fails to build?

**Comment by pgonzal** [comment]
>       "dependencies": {

What's all this then?

---
Refers to: common/config/rush/npm-shrinkwrap.json:14 in 2b9abd6. [](commit_id = 2b9abd64fc085078c22040baa2bb8ca86827b3e6, deletion_comment = False)

File: core-build/test-web-library-build/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
   "devDependencies": {
     "@types/chai": "3.4.34",
     "chai": "~3.5.0",
-    "@types/jest": "~20.0.4",
+    "@types/jest": "~21.1.8",
```
Comment (pgonzal) [inline core-build/test-web-library-build/package.json]: >"~21.1.8" [](start = 19, length = 9)

do you need the PNPM workaround in this PR?

File: apps/api-documenter/package.json (hunk 1 above)
Comment (nickpape-msft) [reply apps/api-documenter/package.json]: Yes!

File: core-build/test-web-library-build/package.json (hunk 2 above)
Comment (nickpape-msft) [reply core-build/test-web-library-build/package.json]: Yes, included

---
In reply to: [159781970](https://github.com/Microsoft/web-build-tools/pull/466#discussion_r159781970) [](ancestors = 159781970)

**Comment by nickpape-msft** [comment]
>       "dependencies": {

The old version of gulp-core-build is apparently using the older jest. Once this is published, the next PR will upgrade the cyclic version of GCB that we are using to be the one we just published, which should make this go away

---
In reply to: [355429219](https://github.com/Microsoft/web-build-tools/pull/466#issuecomment-355429219) [](ancestors = 355429219)

---
Refers to: common/config/rush/npm-shrinkwrap.json:14 in 2b9abd6. [](commit_id = 2b9abd64fc085078c22040baa2bb8ca86827b3e6, deletion_comment = False)

**Comment by pgonzal** [comment]
Please do appropriate testing

**Pull request merged**